modules/proc_container-security-operator-setup.adoc

```diff
@@ -3,7 +3,7 @@
 
 Using the link:https://operatorhub.io/operator/container-security-operator[Container Security Operator],
 (CSO) you can scan container images associated
-with active pods, running on OpenShift and other Kubernetes
+with active pods, running on OpenShift (4.3 or later) and other Kubernetes
```

Comment:
change to "4.2 or later"
release note will indicate variations in functionality

Reply:
Okay.